bundle/edu.gemini.horizons.api/build.sbt

```diff
@@ -18,13 +18,10 @@ osgiSettings
 
 ocsBundleSettings
 
-OsgiKeys.bundleActivator := Some("edu.gemini.horizons.api.osgi.Activator")
```

Comment:
Maybe this should be `None`, otherwise it could pick a default? I don't remember exactly though